src/Cake.NuGet/Cake.NuGet.csproj

```diff
@@ -26,6 +26,7 @@
     <PackageReference Include="NuGet.Frameworks" Version="4.3.0" />
     <PackageReference Include="NuGet.Versioning" Version="4.3.0" />
     <PackageReference Include="NuGet.ProjectModel" Version="4.3.0" />
+    <PackageReference Include="System.ValueTuple" Version="4.4.0" />
```

Comment:
We don't want to take on this dependency at this point.

Reply:
Alright, will change this 👍